.azure-ci/docker_scripts.sh

```diff
@@ -23,7 +24,7 @@ tar -zxvf /boost_1_69_0.tar.gz
 mkdir boost
 cd /boost_1_69_0
 ./bootstrap.sh --prefix=/boost
-./b2 install -j3
+./b2 install -j3 || echo "Parts of boost failed to build. Continuning.."
```

Comment:
This prints the above sentence if boost build exits with a non null exit status.

Reply:
Thanks! Just a small typo in "Continuning.."